python/paddle/v2/framework/op.py

```diff
@@ -216,38 +155,57 @@ def __impl__(*args, **kwargs):
         opdesc = method(*args, **kwargs)
         return core.Operator.create(opdesc.SerializeToString())
 
-    __impl__.__doc__ = get_docstring_from_op_proto(op_proto)
-    __impl__.all_input_args = [var.name for var in op_proto.inputs]
-    __impl__.all_output_args = [var.name for var in op_proto.outputs]
-    __impl__.all_attr_args = [attr.name for attr in op_proto.attrs]
-    __impl__.all_not_temp_output_args = [
-        var.name for var in op_proto.outputs if not var.temporary
-    ]
+    return {
```

Comment:
maybe it's better to have a inner class
```python 
class OpWrapper {
   def __init__(self, op_proto):
      self.op_proto = op_proto
   def inputs():
      return self.op_proto.inputs
    ...
}
```
the benefit is it will have a type, and we can use 
```python
self.get_op_creation_info(type).inputs()
```
to get inputs instead of
```python
self.get_op_creation_info(type)["inputs"]
```
this ask coder to search the code for the dict defination